src/components/Cover/Cover.module.css

```diff
@@ -0,0 +1,95 @@
+.wrapper {
+    text-decoration: none;
+    min-height: 200px;
```

Comment:
I do not see why we need it with the margins inside. If you will remove them you might have it, but this margins the element knows its height

Reply:
removed `min-height`